### PR TITLE
RD-1552 Move tests for dep. create/update with default schedules to u…

### PR DESCRIPTION
**Pull request by glukhman**
…nittests

### CODE REVIEW

File: rest-service/manager_rest/test/endpoints/test_deployments.py [hunk 1]
```diff
@@ -1069,3 +1069,82 @@ def test_update_attributes_already_set(self):
...
+        _, _, _, deployment = self.put_deployment(
+            blueprint_file_name='blueprint_with_default_schedules.yaml')
+
+        sc1_id = '{}_{}'.format(deployment.id, 'sc1')
+        sc2_id = '{}_{}'.format(deployment.id, 'sc2')
+        sc3_id = '{}_{}'.format(deployment.id, 'sc3')
+
+        sched_ids = [sc['id'] for sc in self.client.execution_schedules.list()]
+        self.assertListEqual([sc1_id, sc2_id], sched_ids)
+        self.assertEquals(
```
Comment (YoniItzhak) [inline rest-service/manager_rest/test/endpoints/test_deployments.py:1096]: Why do you make this assertion here and not in the previous test?

File: rest-service/manager_rest/test/endpoints/test_deployments.py [hunk 2]
```diff
@@ -1069,3 +1069,82 @@ def test_update_attributes_already_set(self):
...
+    def test_update_deployment_with_default_schedules(self):
+        _, _, _, deployment = self.put_deployment(
+            blueprint_file_name='blueprint_with_default_schedules.yaml')
+
+        sc1_id = '{}_{}'.format(deployment.id, 'sc1')
+        sc2_id = '{}_{}'.format(deployment.id, 'sc2')
+        sc3_id = '{}_{}'.format(deployment.id, 'sc3')
+
+        sched_ids = [sc['id'] for sc in self.client.execution_schedules.list()]
+        self.assertListEqual([sc1_id, sc2_id], sched_ids)
```
Comment (YoniItzhak) [inline rest-service/manager_rest/test/endpoints/test_deployments.py:1095]: Isn't this already asserted in the previous test?

File: rest-service/manager_rest/test/endpoints/test_deployments.py [hunk 3]
```diff
@@ -1069,3 +1069,82 @@ def test_update_attributes_already_set(self):
...
+            blueprint_file_name='blueprint_with_default_schedules.yaml')
+
+        sc2_id = '{}_{}'.format(deployment.id, 'sc2')
+        self.client.execution_schedules.delete(sc2_id)
+        new_blueprint_id = 'updated_schedules'
+        self.put_blueprint(
+            blueprint_id=new_blueprint_id,
+            blueprint_file_name='blueprint_with_default_schedules2.yaml')
+        dep_up = self.client.deployment_updates.update_with_existing_blueprint(
+            deployment.id, new_blueprint_id, preview=True)
```
Comment (YoniItzhak) [inline rest-service/manager_rest/test/endpoints/test_deployments.py:1124]: How about taking it out to a function?

File: rest-service/manager_rest/test/endpoints/test_deployments.py (hunk 1 above)
Comment (glukhman) [reply rest-service/manager_rest/test/endpoints/test_deployments.py:1096]: copy error :)

File: rest-service/manager_rest/test/endpoints/test_deployments.py (hunk 3 above)
Comment (glukhman) [reply rest-service/manager_rest/test/endpoints/test_deployments.py:1120]: nah, not reused exactly

**Review by YoniItzhak — APPROVED** [review]

**Pull request merged**